buildtools.bzl

```diff
@@ -0,0 +1,152 @@
+load("@bazel_skylib//lib:new_sets.bzl", "sets")
+load("@bazel_skylib//lib:types.bzl", "types")
+
+_TOOL_NAMES = ["buildifier", "buildozer"]
+_TYPICAL_PLATFORMS = ["darwin", "linux"]
+_TYPICAL_ARCHES = ["amd64", "arm64"]
```

Comment:
```suggestion
_TYPICAL_ARCHES = ["amd64", "x86_64"]
```

Reply:
I am looking at the [latest release assets for buildtools](https://github.com/bazelbuild/buildtools/releases/tag/4.2.5). The arches that I see listed are `amd64` and `arm64`. Do you see something different?